learn/appchains/tx-fees.md

```diff
@@ -0,0 +1,116 @@
+
+Under the hood, for execution time, instead of working with a gas-based mechanism, all Tanssi Appchains work with a [weight-based mechanism](https://docs.substrate.io/build/tx-weights-fees/){target=\_blank}. Weight refers to the time (in picoseconds) it takes to validate a block. Generally speaking, for both EVM and non-EVM Tanssi Appchains, all function calls have a weight associated with them, which sets limits on storage input/output and computation. For EVM Appchains, there is a gas-to-weight mapping that fully complies with the expected gas requirements for Ethereum API-based tools.
+
+A transaction fee scheme is applied on top of the weight-based mechanism to ensure economic incentives are in line to limit the execution time, computation, and number of calls (database read/writes) to perform operations. Transaction fees are fundamental to preventing network spam, as they represent the cost of using the Appchain service. Consequently, a user interacting with the network through a specific function call will pay a transaction fee by a baseline fee algorithm.
+
+This section covers the fundamentals of transaction fees for Tanssi Appchains. It first covers the underlying transaction fee architecture and how it is adapted to a fully EIP-1559-compliant model for EVM Appchains.
+
+
+## Baseline Fees Calculation {: #baseline-fees }
+
+Any call that modifies the state of an Appchain has a transaction fee associated with it. This fee is usually associated with the execution cost of the transaction, like _gas_ and _gasPrice_ in EVM-compatible chains. 
+
+Tanssi Appchains use a weight-based mechanism for fees of the [modular framework](/learn/framework/){target=\_blank} also accounts for storage input/outputs. However, Tanssi EVM-compatible Appchains support legacy and [EIP-1559 compatible](https://eips.ethereum.org/EIPS/eip-1559){target=\_blank} transaction pricing mechanisms.
+
+This section outlines all the different concepts associated with transaction fees for Tanssi Appchains.
+
+### Weight {: #baseline-weight}
+
+Broadly speaking, weight refers to the execution time it takes to validate a block, measured in picoseconds. Weight is divided into two separate variables:
+
+- **refTime** - corresponds to the weight associated with computation time and database reads/writes
+- **proofSize** - corresponds to the weigth associated to the size of the _Proof-Of-Validity_ (or _PoV_ for short). The _PoV_ is associated with the relevant state of a transaction, and it is what the Appchain block producer shares with the relay chain validator to get its block finalized as part of the [Appchain transaction flow](/learn/appchains/overview/#appchain-transaction){target=\_blank}
+
+To find the weights for all function calls, they are benchmarked in a system with reference hardware, and the approximate values of _refTime_ and _proofSize_ are set. This process is repeated for all function calls that consume blockspace and affect the _PoV_.
```

Comment:
Same comment here as with gasPrice, we should use `refTime` and `proofSize` if referring to code.

Also, I'm not really used to using italics, but if we're going to use them, we should have a general rule about it. Like are we italicizing words when we first introduce them, or if they're keywords are we going to italicize them everywhere we see them? I think I probably lean more towards using it for introducing the word/concept, but I don't see the purpose in continuing to emphasize them

Reply:
Umm unsure what is the best approach. When doing Research papers in PhD you use italics when talking about variables.

What would be our current standard? not using italics at all?